x/metadata/keeper/keeper_test.go

```diff
@@ -226,40 +227,73 @@ func (s *KeeperTestSuite) TestValidateAllOwnerPartiesAreSigners() {
 			msgTypeURL: "",
 			errorMsg:   "missing signature from [owner (PARTY_TYPE_OWNER)]",
 		},
-		// authz test cases
-		"two parties - one missing signature with authz grant - two signers": {
+	}
+
+	// Test cases
+	for n, tc := range cases {
+		s.T().Run(n, func(t *testing.T) {
+			err := s.app.MetadataKeeper.ValidateAllPartiesAreSignersWithAuthz(s.ctx, tc.owners, tc.signers, tc.msgTypeURL)
+			if len(tc.errorMsg) == 0 {
+				assert.NoError(t, err, "%s unexpected error", n)
+			} else {
+				assert.EqualError(t, err, tc.errorMsg, "%s error", n)
+			}
+		})
+	}
+}
+
+func (s *KeeperTestSuite) TestValidateAllOwnerPartiesAreSignersWithGenericAuthorization() {
```

Comment:
Can we put `TestValidateAllOwnerPartiesAreSigners` back the way it was before? That makes it easier to see that all the previous tests still pass, and that dual breaking changes (one in the main code, one in a test) weren't accidentally introduced.

Reply:
I mean put them back to what they were previously (i.e. no changes at all to those two functions).